examples/generateBlocks/epoch-reached.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+from multiversx_sdk_network_providers import ProxyNetworkProvider
+
+SIMULATOR_URL = "http://localhost:8085"
+GENERATE_BLOCKS_UNTIL_EPOCH_REACHED_URL = f"{SIMULATOR_URL}/simulator/generate-blocks-until-epoch-reached"
+NETWORK_STATUS_URL = f"{SIMULATOR_URL}/network/status/4294967295"
+
+
+def main():
+    # create a network provider
+    provider = ProxyNetworkProvider(SIMULATOR_URL)
+
+    # generate blocks until we reach the epoch 10
+    provider.do_post(f"{GENERATE_BLOCKS_UNTIL_EPOCH_REACHED_URL}/10", {})
```

Comment:
Could have been a constant. But perfectly fine as it is, in this context.

Reply:
added const here: https://github.com/multiversx/mx-chain-simulator-go/pull/22